webapp/src/js/enketo/medic-xpath-extensions.js

```diff
@@ -0,0 +1,16 @@
+var zscoreUtil;
+
+module.exports = {
+  init: function(_zscoreUtil) {
+    zscoreUtil = _zscoreUtil;
+  },
+  func: {
+    'z-score': function(chartId, sex, x, y) {
+      var result = zscoreUtil(chartId, sex, x, y);
+      if (!result) {
+          return { t: 'str', v: '' };
+      }
+      return { t: 'num', v: result };
+    }
+  }
+};
```

Comment:
IIRC there are a couple of other medic-specific extensions that should be moved in here from `openrosa-xpath-extensions`.

Reply:
I have no idea how to find what's medic specific. Did we fork it, or copy it from somewhere?

I'll merge this as is to get a beta out but could you please raise an issue with more detail about what should be moved into this file?